src/main/java/cc/hyperium/Hyperium.java

```diff
@@ -232,6 +232,7 @@ public void init(InitializationEvent event) {
 
             SplashProgress.setProgress(7, I18n.format("splashprogress.startinghyperium"));
             LOGGER.info("[Hyperium] Started!");
+            System.out.println("Running from main class: " + Hyperium.class.getName());
```

Comment:
I don't see why this is necessary. There's no other way it could be running. Not a helpful log output.